subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_df.c

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdint.h>
+#include <errno.h>
+#include <devicetree.h>
+#include <sys/util_macro.h>
+#include <nrfx/hal/nrf_radio.h>
+
+#include "radio_df.h"
+
+/* @brief Minimum antennas number required if antenna switching is enabled */
+#define DF_ANT_NUM_MIN 2
+/* @brief Value to set antenna GPIO pin as not connected */
+#define DF_GPIO_PIN_NOT_SET 0xFF
+/* @brief Number of PSEL_DFEGPIO registers in Radio peripheral */
+#define DF_PSEL_GPIO_NUM 8
+
+/* @brief Direction Finding antenna matrix configuration */
+struct df_ant_cfg {
+	uint8_t ant_num;
+	/* Selection of GPIOs to be used to switch antennas by Radio */
+	uint8_t dfe_gpio[DF_PSEL_GPIO_NUM];
+};
+
+#define DFE_ANT DT_NODELABEL(dfe_ant)
+#define DFE_GPIO_PIN(idx)                                               \
+	COND_CODE_1(DT_NODE_HAS_PROP(DFE_ANT, dfegpio##idx##_pin),      \
+		    (DT_PROP(DFE_ANT, dfegpio##idx##_pin)),             \
+		    (DF_GPIO_PIN_NOT_SET))
+
+#define DFE_GPIO_PIN_DISCONNECT (RADIO_PSEL_DFEGPIO_CONNECT_Disconnected << \
+				 RADIO_PSEL_DFEGPIO_CONNECT_Pos)
+
+#define DFE_GPIO_NUM        (0 UTIL_LISTIFY(DF_PSEL_GPIO_NUM, COUNT_GPIO, _))
```

Comment:
`COUNT_GPIO` is not defined now. You need to restore one line removed in the last update.

Reply:
Done, thank you very much for spotting that